scapy/arch/pcapdnet.py

```diff
@@ -156,9 +156,21 @@ def datalink(self):
           return pcap_datalink(self.pcap)
       def fileno(self):
           if sys.platform.startswith("win"):
-            log_loading.error("Cannot get selectable PCAP fd on Windows")
+            log_loading.error("Cannot get selectable PCAP fd on Windows. "
+                              "Use L2socket/L3socket select() function instead !")
             return 0
-          return pcap_get_selectable_fd(self.pcap) 
+          return pcap_get_selectable_fd(self.pcap)
+      def select(self, socket, timeout):
```

Comment:
This should not be called directly, as located in `_PcapWrapper_pypcap`. The good way would be through `supersocket.py` I'll add comments thoug